Blake Fletcher - Javascript-I

blkfltchr opened the pull request:
First commit
Completed objects.js

emmadrueke commented:
Collaborated with Dre'Sean Williams, Austin Howes, Jackee Rohrich, Vinnie Scalco.

Commit:
Blake Fletcher - JJavascript-I

First commit
Completed objects.js

Commit:
Blake Fletcher - Javascript I

Second commit
Completed challenges 1, 2 and 4
Completed challenges 5 and 6 using filter
To refactor 5 and 6 with for loops
To complete challenge 3

Commit:
Changes

Completed arrays.js challenge #3

Commit:
Final push

Completed challenges 5 and 6 without filter
